ui/src/utils/constants.ts

```diff
@@ -46,6 +47,8 @@ export const ROOT_PATH = "/";
 
 export const SIDER_WIDTH = 320;
 
+export const TO_DEVELOP = "To develop";
```

Comment:
This has never been necessary, and I would urge us not to start doing this, for the following reasons;

1. It's a useless constant in production
2. It's phrased awkwardly
3. It complicates development by having to keep track of a WIP flag
4. There are plenty of unimplemented parts that do not have this flag and adding it only in a few places makes things arbitrary and inconsistent
5. This is a FOSS repo, and having non-functional bits like this makes development confusing and inconsistent

What's wrong with just omitting this altogether? If it's not implemented, it simply isn't, functions and notices don't need to say this explicitly. @amonsosanz, thoughts?

Reply:
I'm ok removing this.